packages/web/src/containers/wallet-balance-container/WalletBalanceContainer.tsx

```diff
@@ -52,33 +68,52 @@ const WalletBalanceContainer: React.FC = () => {
     changeTokenDeposit(undefined);
     if (!address) return;
   }, [connected, address, changeTokenDeposit]);
-
   const withdraw = useCallback(() => {
     if (!connected) return;
     setIsShowWithDrawModal(true);
     if (!address) return;
   }, [connected, address]);
 
-  const claimAll = useCallback(() => {}, []);
-
-  const { displayBalanceMap, updateBalances } = useTokenData();
-  const { positions, loading: loadingPositions } = usePositionData();
-
-  useEffect(() => {
-    setLoadingBalance(true);
-    updateBalances().finally(() => {
-      setLoadingBalance(false);
+  const claimAllReward = useCallback(() => {
+    const data = {
+      tokenASymbol: positions[0]?.pool?.tokenA?.symbol,
+      tokenBSymbol: positions[0]?.pool?.tokenA?.symbol,
+      tokenAAmount: "0.12",
+      tokenBAmount: "0.13",
```

Comment:
Is this number only available statically?

Reply:
This is the old version. I had a discussion with the team and updated the new